maintainers/maintainer-list.nix

```diff
@@ -9203,4 +9203,10 @@
     github = "deifactor";
     githubId = 30192992;
   };
+  koozz = {
```

Comment:
:abcd: Users are mostly inserted by alphabetical order. Could you find the next best spot for your username.

Reply:
I see, the last few were totally unordered and that made me think it was chronological. I've put myself in the ordered part now.